tests/travis/run_test.sh

```diff
@@ -47,24 +47,25 @@
 if [ ${TASK} == "cmake_test" ]; then
     set -e
 
+    CMAKE_COMMON_FLAGS='-DOE_DEBUG=1 -DSIMULATE=ON -DLVI_MITIGATION=OFF'
```

Comment:
Any reason we disable LVI mitigation for all build tests?

Reply:
Currently, it requires some messing around with the OE SDK installation -- it's not enough to simply install the latest SDK; there are a whole bunch of other steps required for installing and using the LVI mitigated versions of the OE libraries.
I'm hoping that the OE maintainers will simplify the build process in a future version of the SDK. Perhaps we can hold off on this until then?

Reply:
Makes sense!